IO/Ex1.hpp

```diff
@@ -0,0 +1,24 @@
+
+#include <fstream> //ios_base::openmode
+#include <string>
+
+class FileHandler {
+public:
+    explicit FileHandler(const std::string path, std::ios_base::openmode permissions); //Ctor
+    FileHandler(const FileHandler &other); //CCtor
+    FileHandler(FileHandler &&other) noexcept; //MCtor
+
+    FileHandler& operator=(const FileHandler &other); //Copy Assignment
+    FileHandler& operator=(FileHandler &&other) noexcept; //Move Assignment
+
+    std::fstream &getFD();
+    static void createFile(const std::string filename);
```

Comment:
How is this function aligned with the rest of this class? What is the relation between the current file opened in the constructor and the new file created by this function?

Reply:
it is a static function that helps with creating files without making the user to cease control on them.
but tell me what you think if it is better to make this method private and just call it at the Ctor.